README.md

```diff
@@ -102,7 +102,47 @@ A preview of the VDK CLI commands:
 
 
 # Getting Started
-Installing VDK is a simple pip command. See the [Getting Started](https://github.com/vmware/versatile-data-kit/wiki/getting-started) guide to install VDK and create a data job.
+
+## Vdk-core and CLI
+
+```
+pip install vdk-core
+```
+
+This installs the core vdk package and the vdk command line interface. You can
+use them to run jobs in your local shell environment.
+
+## Running with docker, helm, and k8s
+
+**Prerequisites**
+- [helm](https://helm.sh/docs/intro/install)
+- [docker](https://docs.docker.com/get-docker)
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) (version 0.11.1 or later)
+
+```
+pip install quickstart-vdk
+vdk server --install
+```
+
+This installs the necessary python modules and runs the vdk server.
+The server includes:
+- control service - the vdk backend, responsible for creating, deploying,
+  scheduling and general management of data jobs
+- operations ui - the vdk user interface
+- git server - stores data job source code
+- docker registry - stores data job images
```

Comment:
Those are impelmeentation details. Our users should not need know they even exist if possible

Reply:
I agree. I think that cloud deployment explained may be extracted to a separate page, being relatively heavier in content to `pip install quickstart-vdk`, that is very much appreciated on the front page. Great idea!

Reply:
@tozka I'll summarise my arguments for leaving the description in the docs. If you still feel strongly about it, I'll remove it.

1. People need to know what they're installing on their system. As a user, maybe I don't want a specific docker container eating all my RAM. If I found out said container wasn't mentioned in the docs, but was deployed anyway, I'd be extremely annoyed.
2. These implementation details are visible anyway. What gets spun up is mentioned in the logs when you run `vdk server --install`. It's also visible in the Docker Desktop UI. It makes sense to put it in the docs and avoid surprises. It's not like we're walking through the complete lifecycle of a data job and the k8s pods required to run it.
3. Anyone willing to install docker, helm and k8s will either have cursory knowledge about those tools, or acquire it during the process.

"Don't make me think" is a book about web design and usability. Some of it might apply to documentation, but certainly not all of it and it definitely should not be viewed as dogma.

Reply:
>  As a user, maybe I don't want a specific docker container eating all my RAM...
You and I are technical people and we tend to want to know those details. But we need to be careful and think not about what I would like but what the users would like.

Including or not those 2 sentences right now is really of no importance. What is far more important is for you to understand my point . Our primary goal of our product documentation should be to guide the user through the necessary steps to get the product or feature working (aka achieve their goal). Including too many details that are not absolutely needed for that could complicate the process and overwhelm users, especially those who are not as technically inclined.

Although some users will have knowledge about Docker, Helm, and Kubernetes, our tool is designed to be used by a wide range of users, not all of whom will have this knowledge or want to acquire it. Frankly, most data engineers are not as technically inclined in my experience. Therefore, the focus should be on ensuring that anyone can use the tool effectively, regardless of their technical background.

I would prefer if we can find a way for users to not need to install kind and helm and further reduce requirements toward the user.

--- 
Also , of course, no book or guideline should ever be viewed as dogma, and the advices given there should be interpreted and applied in light of the specific circumstances or context at hand. Which I find appropriate in this context now hence I recommended it.
The book is more about "advanced common sense" for UX , using examples from web and mobile design domain. 
advices are very applicable for other areas. I can also recommend "[The Design Of Everyday Things](https://www.goodreads.com/en/book/show/840)" which gives very similar advices but uses "everyday things" as an examples instead. I think for myself and also our team has still a lot to learn about good UX and we should not disregard those advices easily.